source/refined-github.css

```diff
@@ -35,11 +35,6 @@
 	display: none !important;
 }
 
-/* Hide `New pull request button` near file list */
-.file-navigation .new-pull-request-btn {
-	display: none;
-}
-
```

Comment:
Cannot attach screenshots to commits so here's a review comment instead.

The "New pull request button" used to be like:

![](https://user-images.githubusercontent.com/8728954/78605776-2bfa3980-782a-11ea-8b41-23bb1510ea3f.png)

Now:

<img width="828" alt="Screen Shot 2022-02-27 at 13 32 56" src="https://user-images.githubusercontent.com/44045911/155869595-d4460bd8-3d1d-4886-9569-2e51d68d56e2.png">

No need to hide it to make space for other features anymore